spec/views/work_package/show.html.erb_spec.rb

```diff
@@ -37,15 +37,6 @@
                        filename: 'foo.jpg')
   }
 
-  it 'renders correct image paths in journal entries' do
-    work_package.add_journal(work_package.author, 'bar !foo.jpg! bar')
-    work_package.attachments << attachment
-    work_package.save
-    render 'history', work_package: work_package, journals: work_package.journals
-    expect(rendered)
-      .to have_selector "img[src='/attachments/#{attachment.id}/download']"
-  end
-
   context 'watchers list is sorted alphabeticaly' do
```

Comment:
If it is not to much effort, I would move this to somewhere else. It was never a test for work_packages/show to begin with and now this is looking really weird.